src/wallet/rpcwallet.cpp

```diff
@@ -909,7 +927,10 @@ UniValue sendmany(const JSONRPCRequest& request)
             "      \"address\"          (string) Subtract fee from this address\n"
             "      ,...\n"
             "    ]\n"
-            "\nResult:\n"
+            "6. opt_in_rbf             (boolean, optional) Allow this transaction to be replaced by a transaction with higher fees\n"
+            "7. conf_target            (numeric, optional) Confirmation target (in blocks)\n"
+            "8. conservative_estimate  (boolean, optional) Use conservative (potentially higher) fee estimation\n"
```

Comment:
I'd prefer to use a string here (which maps to FeeEstimateMode) instead of a boolean. This would also allow adding additional fee estimation strategies in the future without changing the interface.